packages/@aws-cdk/aws-stepfunctions-tasks/test/athena/start-query-execution.test.ts

```diff
@@ -17,7 +17,7 @@ describe('Start Query Execution', () => {
       },
       resultConfiguration: {
         encryptionConfiguration: { encryptionOption: EncryptionOption.S3_MANAGED },
-        outputLocation: 'https://s3.Region.amazonaws.com/bucket-name/key-name',
+        outputLocation: 's3://query-results-bucket/folder/',
```

Comment:
is this change related to the changes being made in this PR? or is it just an unrelated improvement to update the input to something that is valid?

Reply:
Update the input to something valid. The example in `lib/athena/start-query-execution.ts` is valid and should be used in the unit test.

Reply:
got it, so it's not related to this change.